lib/mbaasRequest/mbaasRequest.js

```diff
@@ -158,7 +158,7 @@ function _buildAdminMbaasParams(params) {
 function responseHandler(params, cb) {
 
   return function handleResponse(err, httpResponse, body) {
-    body = body || "{}";
+    body = typeof body === undefined ? "{}" : body;
```

Comment:
I am returning a boolean here so needed to update the check. Boolean is valid JSON so should be allowed